libs/npress/AdminModule/templates/Pages/edit.latte

```diff
@@ -71,31 +71,31 @@
         <span class="add-on"><i class="icon-wrench"></i></span
 				>{$presenter['pageEditForm']['seoname']->control}
 			</div>
-			<p class="help-inline"><a href="{$page->link()}" rel="tooltip" title="{$page->link(true)}" class="btn btn-success">zobrazit na webu</a>
+			<p class="help-inline"><a href="{$page->link()}" rel="tooltip" title="{$page->link(true)}" class="btn btn-success">prikazati na web sajtu</a>
```

Comment:
web sajtu? zašto ne "web stranici" ili samo "stranici" ili "veb sajtu" web siteu? odnosno bilo šta drugo

Reply:
Odnosi se na uređivanje stranice (web page) koja se dodaje na website, tako da bi ispalo "prikazati stranicu na stranici". Da stavim samo "sajt", bez "web"?

Reply:
kontam, kontam. meni se nije svidjela ova kombinacija originalne i transkribovane riječi. može samo "na sajtu"